index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <link rel="stylesheet" href="style.css">
+
+    <title>Jonathan Sivahera</title>
+  </head>
+  <body>
+    <div class="blur-item circle">
+      
+    </div>
+    <!-- Header section -->
+    <header class="header-section">
+    <a href="index.html#home">JS</a>
+    <div class="hamburger"></div>
+    </header>
+    <!-- Main page content -->
+    <main class="main-section" id="home">
```

Comment:
In our requirements for the project, this line should have a `section` tag rather than `main`. Your usage of main tag is perfect 🔥 but kindly change the element tag to section one to meet the requirements. 

Another alternative if you wish to keep `main` would be to use a `section` tag inside the main!